index.html

```diff
@@ -1265,9 +1265,7 @@
 
 [=User agents=] should aggressively <a href="#data-minimization">minimize</a> [=ancillary
 data=] and should avoid burdening the user with additional [=privacy labor=]
-when deciding what [=ancillary data=] to expose. To that end, user agents may
-employ user research, solicitation of general preferences, and heuristics about
-sensitivity of data or trust in a particular context. To facilitate site
+when deciding what [=ancillary data=] to expose. To facilitate site
 understanding of user preferences, user agents can provide browser-configurable
```

Comment:
Should "browser-configurable" be "user-configurable" here?

Reply:
browser-configurable here is the same thing as user-configureable.  "browser-configureable" means the user can configure the browser to change the policy